bin/start-func-test-server.js

```diff
@@ -121,6 +121,8 @@ new Promise((resolve) => {
       `NODE_APP_INSTANCE=${appInstance}`,
       '-e',
       `NODE_ENV=${nodeEnv}`,
+      '-e',
+      `USE_HTTPS=true`,
```

Comment:
I think you should use `-e` here to hard code everything to `https://example.com:4000`. You can follow [this example](https://github.com/mozilla/addons-frontend/#configuring-an-android-device-for-local-development) but you won't need to set `WEBPACK_SERVER_HOST`.